src/js/themes/colors.js

```diff
@@ -37,10 +37,13 @@ export const colors = {
     light: '#000000',
   },
   'text-weak': {
+    dark: '#8C98AA',
```

Comment:
Do we need to change places that talk about using this for disabled like `disabled-text` (line 62) and `status-disabled` (line 81) (not to mention I always wondered why these were deprecated :) )

Reply:
Let me look into that

Reply:
Here's background:

`status-disabled` was never being used in designs, so we just deprecated it in favor of `text-weak` / `border-weak`: https://github.com/grommet/grommet-theme-hpe/pull/138
`disabled-text` was deprecated to keep more semantic alignment in the "steps" of text colors: https://github.com/grommet/grommet-theme-hpe/pull/133 -- however, I could see an advantage now that we have  a new `text-weak` to naming the new `text-xweak` as `text-disabled` as opposed to `text-xweak` so it's clear when to use that text color (since it won't meet color contrast requirements for readability). but I could go either way here.

Reply:
`text-xweak` aligns more with the grommet pattern. I could see defining `disabled-text` to be `text-xweak` to add a layer of indirection.

Reply:
I'm good just leaving `text-xweak` in that case and not doing any specific disabled-text.

Reply:
@ericsoderberghp would you agree with changing `status-disabled` and `disabled-text` I can do that in this PR or if we want to discuss a little more I can make another issue

Reply:
Let's handle that as a separate issue, to ensure we add it consistently to grommet.